ci/concourse-defaults.yml

```diff
@@ -1,12 +1,6 @@
 postfix-config-branch: master
 postfix-config-uri: https://github.com/cloud-gov/cg-deploy-postfix
 
-# slack-webhook-url: XXX
-slack-username: concourse
```

Comment:
This one I think you need

Reply:
There is a Credhub variable that supplies this value, so we don't need to define it manually